src/main/java/com/lambdaschool/javaorders/model/Order.java

```diff
@@ -0,0 +1,102 @@
+package com.lambdaschool.javaorders.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "orders")
+public class Order
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long ordnum;
+
+    private double ordamount;
+    private double advanceamount;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "custcode", nullable = false)
+    @JsonIgnoreProperties({"agent", "orders"})
+    private Customer customer;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "agentcode", nullable = false)
+    @JsonIgnoreProperties({"customer", "orders"})
+    private Agent agent;
```

Comment:
Line 25 - typo on one of your properties (the `Agent` object has plural **`customers`**).

Also, since you are using lazy fetching, you'll want to ignore the automatically added `hibernateLazyInitializer` property as well.